demo/guide-python/distributed_extmem_basic.py

```diff
@@ -13,8 +13,9 @@
 If `device` is `cuda`, following are also needed:
 
 - cupy
-- python-cuda
 - rmm
+- pynvml (optional)
```

Comment:
Although the Python module is still named `pynvml`, the package has been deprecated ([see warning at the top](https://github.com/gpuopenanalytics/pynvml)). I would suggest instead documenting the new package name [nvidia-ml-py](https://pypi.org/project/nvidia-ml-py/).

Reply:
Done.